src/features/Dashboard/components/helpers.ts

```diff
@@ -0,0 +1,95 @@
+import approval from '@assets/images/transactions/approval.svg';
+import contractDeploy from '@assets/images/transactions/contract-deploy.svg';
+import contractInteract from '@assets/images/transactions/contract-interact.svg';
+import defizap from '@assets/images/transactions/defizap.svg';
+import inbound from '@assets/images/transactions/inbound.svg';
+import membershipPurchase from '@assets/images/transactions/membership-purchase.svg';
+import outbound from '@assets/images/transactions/outbound.svg';
+import swap from '@assets/images/transactions/swap.svg';
+import transfer from '@assets/images/transactions/transfer.svg';
+import { translateRaw } from '@translations';
+import { Asset, ITxTypeMeta } from '@types';
+
+import { ITxHistoryType } from '../types';
+import { ITxTypeConfigObj } from './RecentTransactionList';
+
+export const constructTxTypeConfig = ({ type, protocol }: ITxTypeMeta): ITxTypeConfigObj => ({
```

Comment:
Test please.

Reply:
Is tested in https://github.com/MyCryptoHQ/MyCrypto/pull/4060/files#diff-c094c91a2357ce7598aa85b367b25fb5f4d79f287be1a914d4af52d5eef376ddR1-R31